_rules/aria-attr-defined-5f99a7.md

```diff
@@ -78,15 +78,15 @@ Multiple valid ARIA 1.1 attributes `aria-*` are specified on element `input` wit
 
 #### Failed Example 1
 
-`aria-not-checked` is not a defined attribute in ARIA 1.1.
+This `li` element with a role of `menuitemcheckbox` has an `aria-*` attribute which is not defined in [WAI-ARIA Specifications][].
 
 ```html
 <li role="menuitemcheckbox" aria-not-checked="true">List Item</li>
 ```
 
 #### Failed Example 2
 
-`aria-labelled` is not a defined attribute in ARIA 1.1.
+This `div` element with a role of `searchbox` has an `aria-*` attribute which is not defined in [WAI-ARIA Specifications][].
```

Comment:
```suggestion
This `div` element with a role of `searchbox` has an `aria-*` attribute (`aria-labelled`) which is not defined in [WAI-ARIA Specifications][].
```
Because the other one (`aria-placeholder`) is OK.